src/libs/Launcher.js

```diff
@@ -289,6 +289,27 @@ export default class Launcher {
       trigger = triggerResponse.data
       result.createdTrigger = trigger
       log.debug(`ensureAccountAndTriggerAndJob: created trigger`, trigger)
+      // @ts-ignore
```

Comment:
can you add a comment to explain why you need to ignore 🙏

Reply:
Done, the message attribute of IOCozyTrigger is kind of a free object. Its content depends on the kind of trigger. But all the konnector trigger (client or node) which have a destination folder will have the `.message.folder_to_save` attribute.

Reply:
So let's update the IOCozyTrigger type on Cozy-client 🙏